.gitignore

```diff
@@ -1,2 +1,3 @@
+*.css
```

Comment:
You're adding a .css in this PR.  Is the intent to only check in the source scss files and then generate the .css files as part of the build/deploy process?

Reply:
This is my bad, I believe I staged the file before remembering the gitignore. Yes, the plan is to only check in scss and build css for dev/deploys. I'll remove this css file.

Reply:
FYI, the css and the associated map are still in the PR

Reply:
Just removed these.